### PR TITLE
Simplemob AI Tweaks

### LINKED ISSUE #12008: Simple mobs attack until player death
Maybe its intentional, though as far as I know this is a new behaviour caused by the fact that people don't get knocked out when they're in major pain. So, the simple mobs keep attacking.

Sleeping will cause them to stop still but that's probably pretty gamey.

### PR DESCRIPTION
**Pull request by atteria**
Doubled the callback time on the simplemob dodging. It feels a lot less frustrating.
Made hostile simplemobs targeting discard targets that are paralyzed. This mostly solves the problem, and also prevents mobs from shooting at and really aggressively camping downed targets. (Fixes #12008.)
Interrupts attacks against paralyzed targets, since targeting takes a bit to recalculate.

Doesn't prevent attacks on stunned/weakened opponents.

Tested a bunch by losing to hivebot swarms, works great, feels more fun.

### CODE REVIEW

**Comment by atteria** [comment]
!review

File: code/modules/mob/living/simple_animal/hostile/hostile.dm [hunk 1]
```diff
@@ -75,6 +75,10 @@
 			var/mob/M = A
 			if(M.key && !M.client)
 				continue
+		if(ishuman(A))
+			var/mob/living/carbon/human/H = A
+			if(H.paralysis)
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/simple_animal/hostile/hostile.dm:80]: should check for unconsciousness instead

File: code/modules/mob/living/simple_animal/hostile/hostile.dm [hunk 2]
```diff
@@ -214,6 +218,9 @@
 			resist_grab()
 			return
 	var/atom/target
+	if(ishuman(target_mob))
+		if(target_mob.paralysis)
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/simple_animal/hostile/hostile.dm:222]: ditto

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/hostile.dm (hunk 1 above)
Comment (atteria) [reply code/modules/mob/living/simple_animal/hostile/hostile.dm:80]: From testing, checking paralysis works as intended, checking `sleeping` or `M.stat == UNCONCIOUS` just results in mobs beating the players to death.

**Review by SleepyGemmy — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**